content/user/admin_pages/customers/orders.md

```diff
@@ -26,6 +26,8 @@ To the left of the order id, a red dot indicates that the order's shipping addre
 
 Clicking an order selects it and opens a preview on the right hand side of the page.  From there, the order may be opened or deleted.  The invoice or packingslip for an order may also be opened from the preview. 
 
+In Zen Cart 1.5.7, the preview window also allows you to change the status of the order (and optionally send an email to the customer).  The *Send Email* checkbox is prefilled according to the setting in [Admin > Configuration > My Store > Default for Notify Customer on Order Status Update?](/user/admin_pages/configuration/configuration_mystore/#default_for_notify_customer_on_order_status_update).
```

Comment:
I guess it's inevitable that people will keep putting comments like these which are specific to when a certain feature was added.

Given that it's reasonable to expect that usually most features will be retained in future versions, the wording should not be "In Zen Cart 1.5.7", but rather "Since Zen Cart v1.5.7" ... so that it organically "applies" without having to be updated every time a version is released.

Reply:
I like "Since."